builder/packages_image.go

```diff
@@ -260,17 +263,31 @@ func (p *PackagesImageBuilder) generateDockerfile(baseImage string, packages mod
 
 // GetRolePackageImageName generates a docker image name for the amalgamation for a role image
```

Comment:
Ah. No. This is the `packages layer`. The `role` thing confused me.
Maybe change the whole function name to `GetPackageLayerImageName` ?
And the description would then be
```
GetPackageLayerImageName generates a docker image name for the amalgamation holding the packages used in all roles.
```